gradle.properties

```diff
@@ -0,0 +1,10 @@
+# Magic options for more perf
+org.gradle.jvmargs=-Xmx2048M -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
```

Comment:
Have you tested this with Loom? We *might* want to bump this to 3 or even 4GB. Though I do believe newer Loom versions use significantly less memory. Other than that these changes LGTM

Reply:
Nope! I'm going to test that out real quick, but if it _does_ fail, then I want to find a way to give loom 4gb without giving the _rest_ of the gradle daemons 4gb.